### ISSUE TITLE
BR CI without space

### ISSUE THREAD

**Issue by patrickelectric** [bluerobotics/BlueOS-docker#1547]
The raspberry image needs our own CI to be used for it to be generated
```
873588c08a31: Download complete
7c1c9ff422fd: Verifying Checksum
7c1c9ff422fd: Download complete
d1df34ddbebe: Verifying Checksum
d1df34ddbebe: Download complete
ba233b78e8c4: Download complete
28f9da5f211f: Verifying Checksum
28f9da5f211f: Download complete
62ed989198d5: Verifying Checksum
62ed989198d5: Download complete
89f4ef418459: Download complete
time="2023-03-20T17:04:55.209839194Z" level=info msg="Attempting next endpoint for pull after error: failed to register layer: Error processing tar file(exit status 1): write /usr/lib/arm-linux-gnueabihf/libxvidcore.so.4.3: no space left on device"
failed to register layer: Error processing tar file(exit status 1): write /usr/lib/arm-linux-gnueabihf/libxvidcore.so.4.3: no space left on device
### Error: "bash" returned 1, cleaning up...
```

**Comment by Williangalvani** [comment]
The issues to still be there, even after the pump. It also seems the pump is not actually freeing enough space:
https://github.com/bluerobotics/BlueOS-docker/actions/runs/4581065603/jobs/8099863439#step:4:383

**Comment by patrickelectric** [comment]
Appears that the pump is working: 
![image](https://user-images.githubusercontent.com/1215497/230524573-e4129b77-4fd9-4662-93ed-1675e03b0772.png)

**Comment by patrickelectric** [comment]
Fixed by #1608
